src/advanced-settings/AdvancedSettings.test.jsx

```diff
@@ -122,4 +122,14 @@ describe('<AdvancedSettings />', () => {
       expect(textarea.value).toBe('[]');
     });
   });
+  it('should update the textarea value and display the updated value after clicking "Change manually"', async () => {
+    const { getByLabelText, getByText } = render(<RootWrapper />);
+    await waitFor(() => {
+      const textarea = getByLabelText(/Advanced Module List/i);
+      fireEvent.change(textarea, { target: { value: '[3, 2, 1' } });
+      fireEvent.click(getByText(messages.buttonSaveText.defaultMessage));
+      fireEvent.click(getByText(/Change manually/i));
```

Comment:
Do we need to use translation here and above?

Reply:
Yes, I tried, including through the `data-testid`. But this is impossible, probably due to nesting, the floating of elements, or other things. I propose to stop at this option since we are looking for only one line from the `ModalError` component